roles/bwc/tasks/bwc_repos_setup.yml

```diff
@@ -0,0 +1,42 @@
+---
+
+- name: Assert that master_token is specified
+  fail: msg="License key must be supplied for BWC enterprise installation."
+  when: bwc_license is not defined
+
+- name: Create packagecloud dir
+  become: yes
+  file:
+    path: "/etc/packagecloud"
+    mode: "u=rwx,g=rx,o=rx"
+    owner: st2
+    group: st2
+    state: directory
+
+- name: Get read token for repo from packagecloud
+  become: yes
+  no_log: yes
+  uri:
+    url: https://{{ bwc_license }}:@packagecloud.io/install/repositories/StackStorm/{{ bwc_pkg_repo }}/tokens.text
+    creates: "/etc/packagecloud/StackStorm_{{ bwc_pkg_repo }}_read_token.txt"  # Don't download if file already exists
```

Comment:
Yep. Unfortunately, if I generate a token every time, it would cause idempotency failures. If I do what I did, then installation with fail if they swapped the license. But I think this is fine. It fails out with an error. 

Another approach is to ask user for ``{{ bwc_read_token }}`` which I am -1 to.